Apps/APN-UIKit/APN UIKit/AppDelegate.swift

```diff
@@ -57,11 +57,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         if let cdnHost = storage.cdnHost, !cdnHost.isEmpty {
             config.cdnHost(cdnHost)
         }
-        CustomerIO.initialize(withConfig: config.build())
-        let autoScreenTrack = storage.isTrackScreenEnabled ?? true
-        if autoScreenTrack {
-            CustomerIO.shared.add(plugin: AutoTrackingScreenViews(filterAutoScreenViewEvents: nil, autoScreenViewBody: nil))
+        if storage.isTrackScreenEnabled == true {
+            config.autoTrackUIKitScreenViews()
         }
+        CustomerIO.initialize(withConfig: config.build())
```

Comment:
were we not building config before? 😅

Reply:
We were, I think its just code movement that Github is showing it as deleted and added again 😅